A51-custom-backend-metrics.md

```diff
@@ -78,27 +77,36 @@ function recordRequestCostMetric(String name, double value);
 // Records a utilization metric measurement for the call.
 function recordUtilizationMetric(String name, double value);
 
+// Records an opaque named metric measurement for the call.
+function recordNamedMetricsMetric(String name, double value);
+
 // Records the CPU utilization metric measurement for the call.
 function recordCPUUtilizationMetric(double value);
 
 // Records the memory utilization metric measurement for the call.
 function recordMemoryUtilizationMetric(double value);
 
+// Records the memory utilization metric measurement for the call.
+function recordMemoryUtilizationMetric(double value);
```

Comment:
This is a duplicate of the method above.

Reply:
Fixed.